docs/content.en/docs/release-notes/_index.md

```diff
@@ -12,11 +12,11 @@ Information about release notes of INFINI Framework is provided here.
 ### Breaking changes  
 ### Features  
 - feat: add query query_string and prefix support to orm module
-
+- feat: add compress for http process
```

Comment:
feat: add compression support to HTTP processor

...

fix: reload and notify when `pipeline` config changes are detected